Apr 02, 2018 8:53:28 PM ac.at.ju.storageUnit.CassandraConnector connect
SEVERE: null
com.datastax.driver.core.exceptions.NoHostAvailableException: All host(s) tried for query failed (tried: /10.10.0.240:9042 (com.datastax.driver.core.exceptions.TransportException: [/10.10.0.240:9042] Cannot connect))
	at com.datastax.driver.core.ControlConnection.reconnectInternal(ControlConnection.java:233)
	at com.datastax.driver.core.ControlConnection.connect(ControlConnection.java:79)
	at com.datastax.driver.core.Cluster$Manager.init(Cluster.java:1473)
	at com.datastax.driver.core.Cluster.getMetadata(Cluster.java:399)
	at ac.at.ju.storageUnit.CassandraConnector.connect(CassandraConnector.java:78)
	at ac.at.ju.schemaDesign.CassandraHandling.cassandraConnection(CassandraHandling.java:58)
	at ac.at.ju.storageUnit.DatasetEntry.dataStorage(DatasetEntry.java:37)
	at ac.at.ju.storageUnit.DatasetEntry.mainReadJSONObject(DatasetEntry.java:60)
	at ac.at.ju.storageUnit.DatasetEntry.main(DatasetEntry.java:144)

Apr 02, 2018 8:54:13 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 02, 2018 8:54:13 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 02, 2018 8:54:13 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 02, 2018 8:54:13 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 02, 2018 8:54:13 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 02, 2018 8:54:13 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 02, 2018 8:54:13 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 02, 2018 8:54:13 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 02, 2018 8:54:13 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 02, 2018 8:54:13 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 02, 2018 8:54:13 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 25, 2018 4:58:08 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 25, 2018 4:58:08 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 25, 2018 6:31:44 PM ac.at.ju.storageUnit.CassandraConnector connect
SEVERE: null
com.datastax.driver.core.exceptions.NoHostAvailableException: All host(s) tried for query failed (tried: /10.10.0.241:9042 (com.datastax.driver.core.exceptions.TransportException: [/10.10.0.241:9042] Cannot connect))
	at com.datastax.driver.core.ControlConnection.reconnectInternal(ControlConnection.java:233)
	at com.datastax.driver.core.ControlConnection.connect(ControlConnection.java:79)
	at com.datastax.driver.core.Cluster$Manager.init(Cluster.java:1473)
	at com.datastax.driver.core.Cluster.getMetadata(Cluster.java:399)
	at ac.at.ju.storageUnit.CassandraConnector.connect(CassandraConnector.java:78)
	at ac.at.ju.schemaDesign.CassandraHandling.cassandraConnection(CassandraHandling.java:58)
	at ac.at.ju.storageUnit.DatasetEntry.dataStorage(DatasetEntry.java:37)
	at ac.at.ju.storageUnit.DatasetEntry.mainReadJSONObject(DatasetEntry.java:60)
	at ac.at.ju.storageUnit.DatasetEntry.main(DatasetEntry.java:144)

Apr 25, 2018 6:33:42 PM ac.at.ju.storageUnit.CassandraConnector connect
SEVERE: null
com.datastax.driver.core.exceptions.NoHostAvailableException: All host(s) tried for query failed (tried: /10.10.0.241:9042 (com.datastax.driver.core.exceptions.TransportException: [/10.10.0.241:9042] Cannot connect))
	at com.datastax.driver.core.ControlConnection.reconnectInternal(ControlConnection.java:233)
	at com.datastax.driver.core.ControlConnection.connect(ControlConnection.java:79)
	at com.datastax.driver.core.Cluster$Manager.init(Cluster.java:1473)
	at com.datastax.driver.core.Cluster.getMetadata(Cluster.java:399)
	at ac.at.ju.storageUnit.CassandraConnector.connect(CassandraConnector.java:78)
	at ac.at.ju.schemaDesign.CassandraHandling.cassandraConnection(CassandraHandling.java:58)
	at ac.at.ju.storageUnit.DatasetEntry.dataStorage(DatasetEntry.java:37)
	at ac.at.ju.storageUnit.DatasetEntry.mainReadJSONObject(DatasetEntry.java:60)
	at ac.at.ju.storageUnit.DatasetEntry.main(DatasetEntry.java:144)

Apr 25, 2018 6:53:55 PM ac.at.ju.storageUnit.CassandraConnector connect
SEVERE: null
com.datastax.driver.core.exceptions.NoHostAvailableException: All host(s) tried for query failed (tried: /10.10.0.241:9042 (com.datastax.driver.core.exceptions.TransportException: [/10.10.0.241:9042] Cannot connect))
	at com.datastax.driver.core.ControlConnection.reconnectInternal(ControlConnection.java:233)
	at com.datastax.driver.core.ControlConnection.connect(ControlConnection.java:79)
	at com.datastax.driver.core.Cluster$Manager.init(Cluster.java:1473)
	at com.datastax.driver.core.Cluster.getMetadata(Cluster.java:399)
	at ac.at.ju.storageUnit.CassandraConnector.connect(CassandraConnector.java:78)
	at ac.at.ju.schemaDesign.CassandraHandling.cassandraConnection(CassandraHandling.java:58)
	at ac.at.ju.storageUnit.DatasetEntry.dataStorage(DatasetEntry.java:37)
	at ac.at.ju.storageUnit.DatasetEntry.mainReadJSONObject(DatasetEntry.java:60)
	at ac.at.ju.storageUnit.DatasetEntry.main(DatasetEntry.java:144)

Apr 25, 2018 6:56:06 PM ac.at.ju.storageUnit.CassandraConnector connect
SEVERE: null
com.datastax.driver.core.exceptions.NoHostAvailableException: All host(s) tried for query failed (tried: /10.10.0.241:9042 (com.datastax.driver.core.exceptions.TransportException: [/10.10.0.241:9042] Cannot connect))
	at com.datastax.driver.core.ControlConnection.reconnectInternal(ControlConnection.java:233)
	at com.datastax.driver.core.ControlConnection.connect(ControlConnection.java:79)
	at com.datastax.driver.core.Cluster$Manager.init(Cluster.java:1473)
	at com.datastax.driver.core.Cluster.getMetadata(Cluster.java:399)
	at ac.at.ju.storageUnit.CassandraConnector.connect(CassandraConnector.java:78)
	at ac.at.ju.schemaDesign.CassandraHandling.cassandraConnection(CassandraHandling.java:58)
	at ac.at.ju.storageUnit.DatasetEntry.dataStorage(DatasetEntry.java:37)
	at ac.at.ju.storageUnit.DatasetEntry.mainReadJSONObject(DatasetEntry.java:60)
	at ac.at.ju.storageUnit.DatasetEntry.main(DatasetEntry.java:144)

Apr 25, 2018 6:57:16 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 25, 2018 6:57:16 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 25, 2018 6:57:16 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 25, 2018 6:57:16 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 25, 2018 6:57:16 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 25, 2018 7:05:40 PM ac.at.ju.storageUnit.CassandraConnector connect
SEVERE: null
com.datastax.driver.core.exceptions.NoHostAvailableException: All host(s) tried for query failed (tried: /10.10.0.241:9042 (com.datastax.driver.core.exceptions.TransportException: [/10.10.0.241:9042] Cannot connect))
	at com.datastax.driver.core.ControlConnection.reconnectInternal(ControlConnection.java:233)
	at com.datastax.driver.core.ControlConnection.connect(ControlConnection.java:79)
	at com.datastax.driver.core.Cluster$Manager.init(Cluster.java:1473)
	at com.datastax.driver.core.Cluster.getMetadata(Cluster.java:399)
	at ac.at.ju.storageUnit.CassandraConnector.connect(CassandraConnector.java:78)
	at ac.at.ju.schemaDesign.CassandraHandling.cassandraConnection(CassandraHandling.java:58)
	at ac.at.ju.storageUnit.DatasetEntry.dataStorage(DatasetEntry.java:37)
	at ac.at.ju.storageUnit.DatasetEntry.mainReadJSONObject(DatasetEntry.java:60)
	at ac.at.ju.storageUnit.DatasetEntry.main(DatasetEntry.java:144)

Apr 25, 2018 7:17:16 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 25, 2018 7:17:16 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 25, 2018 7:17:16 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 25, 2018 7:17:16 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 25, 2018 7:17:16 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 25, 2018 8:07:24 PM ac.at.ju.storageUnit.CassandraConnector connect
SEVERE: null
com.datastax.driver.core.exceptions.NoHostAvailableException: All host(s) tried for query failed (tried: /10.10.0.241:9042 (com.datastax.driver.core.exceptions.TransportException: [/10.10.0.241:9042] Cannot connect))
	at com.datastax.driver.core.ControlConnection.reconnectInternal(ControlConnection.java:233)
	at com.datastax.driver.core.ControlConnection.connect(ControlConnection.java:79)
	at com.datastax.driver.core.Cluster$Manager.init(Cluster.java:1473)
	at com.datastax.driver.core.Cluster.getMetadata(Cluster.java:399)
	at ac.at.ju.storageUnit.CassandraConnector.connect(CassandraConnector.java:78)
	at ac.at.ju.schemaDesign.CassandraHandling.cassandraConnection(CassandraHandling.java:58)
	at ac.at.ju.storageUnit.DatasetEntry.dataStorage(DatasetEntry.java:37)
	at ac.at.ju.storageUnit.DatasetEntry.mainReadJSONObject(DatasetEntry.java:60)
	at ac.at.ju.storageUnit.DatasetEntry.main(DatasetEntry.java:144)

Apr 25, 2018 8:15:31 PM ac.at.ju.storageUnit.CassandraConnector connect
SEVERE: null
com.datastax.driver.core.exceptions.NoHostAvailableException: All host(s) tried for query failed (tried: /10.10.0.241:9042 (com.datastax.driver.core.exceptions.TransportException: [/10.10.0.241:9042] Cannot connect))
	at com.datastax.driver.core.ControlConnection.reconnectInternal(ControlConnection.java:233)
	at com.datastax.driver.core.ControlConnection.connect(ControlConnection.java:79)
	at com.datastax.driver.core.Cluster$Manager.init(Cluster.java:1473)
	at com.datastax.driver.core.Cluster.getMetadata(Cluster.java:399)
	at ac.at.ju.storageUnit.CassandraConnector.connect(CassandraConnector.java:78)
	at ac.at.ju.schemaDesign.CassandraHandling.cassandraConnection(CassandraHandling.java:58)
	at ac.at.ju.storageUnit.DatasetEntry.dataStorage(DatasetEntry.java:37)
	at ac.at.ju.storageUnit.DatasetEntry.mainReadJSONObject(DatasetEntry.java:60)
	at ac.at.ju.storageUnit.DatasetEntry.main(DatasetEntry.java:144)

Apr 25, 2018 8:21:09 PM ac.at.ju.storageUnit.CassandraConnector connect
SEVERE: null
com.datastax.driver.core.exceptions.NoHostAvailableException: All host(s) tried for query failed (tried: /10.10.0.241:9042 (com.datastax.driver.core.exceptions.TransportException: [/10.10.0.241:9042] Cannot connect))
	at com.datastax.driver.core.ControlConnection.reconnectInternal(ControlConnection.java:233)
	at com.datastax.driver.core.ControlConnection.connect(ControlConnection.java:79)
	at com.datastax.driver.core.Cluster$Manager.init(Cluster.java:1473)
	at com.datastax.driver.core.Cluster.getMetadata(Cluster.java:399)
	at ac.at.ju.storageUnit.CassandraConnector.connect(CassandraConnector.java:78)
	at ac.at.ju.schemaDesign.CassandraHandling.cassandraConnection(CassandraHandling.java:58)
	at ac.at.ju.storageUnit.DatasetEntry.dataStorage(DatasetEntry.java:37)
	at ac.at.ju.storageUnit.DatasetEntry.mainReadJSONObject(DatasetEntry.java:60)
	at ac.at.ju.storageUnit.DatasetEntry.main(DatasetEntry.java:144)

Apr 25, 2018 8:22:56 PM ac.at.ju.storageUnit.CassandraConnector connect
SEVERE: null
com.datastax.driver.core.exceptions.NoHostAvailableException: All host(s) tried for query failed (tried: /10.10.0.241:9042 (com.datastax.driver.core.exceptions.TransportException: [/10.10.0.241:9042] Cannot connect))
	at com.datastax.driver.core.ControlConnection.reconnectInternal(ControlConnection.java:233)
	at com.datastax.driver.core.ControlConnection.connect(ControlConnection.java:79)
	at com.datastax.driver.core.Cluster$Manager.init(Cluster.java:1473)
	at com.datastax.driver.core.Cluster.getMetadata(Cluster.java:399)
	at ac.at.ju.storageUnit.CassandraConnector.connect(CassandraConnector.java:78)
	at ac.at.ju.schemaDesign.CassandraHandling.cassandraConnection(CassandraHandling.java:58)
	at ac.at.ju.storageUnit.DatasetEntry.dataStorage(DatasetEntry.java:37)
	at ac.at.ju.storageUnit.DatasetEntry.mainReadJSONObject(DatasetEntry.java:60)
	at ac.at.ju.storageUnit.DatasetEntry.main(DatasetEntry.java:144)

Apr 25, 2018 8:24:57 PM ac.at.ju.storageUnit.CassandraConnector connect
SEVERE: null
com.datastax.driver.core.exceptions.NoHostAvailableException: All host(s) tried for query failed (tried: /10.10.0.241:9042 (com.datastax.driver.core.exceptions.TransportException: [/10.10.0.241:9042] Cannot connect))
	at com.datastax.driver.core.ControlConnection.reconnectInternal(ControlConnection.java:233)
	at com.datastax.driver.core.ControlConnection.connect(ControlConnection.java:79)
	at com.datastax.driver.core.Cluster$Manager.init(Cluster.java:1473)
	at com.datastax.driver.core.Cluster.getMetadata(Cluster.java:399)
	at ac.at.ju.storageUnit.CassandraConnector.connect(CassandraConnector.java:78)
	at ac.at.ju.schemaDesign.CassandraHandling.cassandraConnection(CassandraHandling.java:58)
	at ac.at.ju.storageUnit.DatasetEntry.dataStorage(DatasetEntry.java:37)
	at ac.at.ju.storageUnit.DatasetEntry.mainReadJSONObject(DatasetEntry.java:60)
	at ac.at.ju.storageUnit.DatasetEntry.main(DatasetEntry.java:144)

Apr 25, 2018 8:51:38 PM ac.at.ju.storageUnit.CassandraConnector connect
SEVERE: null
com.datastax.driver.core.exceptions.NoHostAvailableException: All host(s) tried for query failed (tried: /10.10.0.242:9042 (com.datastax.driver.core.exceptions.TransportException: [/10.10.0.242:9042] Cannot connect))
	at com.datastax.driver.core.ControlConnection.reconnectInternal(ControlConnection.java:233)
	at com.datastax.driver.core.ControlConnection.connect(ControlConnection.java:79)
	at com.datastax.driver.core.Cluster$Manager.init(Cluster.java:1473)
	at com.datastax.driver.core.Cluster.getMetadata(Cluster.java:399)
	at ac.at.ju.storageUnit.CassandraConnector.connect(CassandraConnector.java:78)
	at ac.at.ju.schemaDesign.CassandraHandling.cassandraConnection(CassandraHandling.java:58)
	at ac.at.ju.storageUnit.DatasetEntry.dataStorage(DatasetEntry.java:37)
	at ac.at.ju.storageUnit.DatasetEntry.mainReadJSONObject(DatasetEntry.java:60)
	at ac.at.ju.storageUnit.DatasetEntry.main(DatasetEntry.java:144)

Apr 25, 2018 9:19:57 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 25, 2018 9:19:57 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 25, 2018 9:19:57 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 25, 2018 9:19:57 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
Apr 25, 2018 9:19:57 PM ac.at.ju.storageUnit.CassandraConnector existKeyspace
SEVERE: null
